wordpress-mobile/WordPress-iOS
# Fix spacing and font size of the message displayed when there are no domain suggestions during search

Fixes [#15155](https://github.com/wordpress-mobile/WordPress-iOS/issues/15155) with additional enhancements.

### Description

This PR fixes a few UI issues we observed **on the error message** that is displayed when a domain search returns no domain suggestions results:

- Adds a padding to the error message to make sure it doesn't reach the horizontal edges of the screen
- Set the font size smaller to make it fit better with the style of the page.
- Fix the visibility of the message in landscape orientation when the keyboard is displayed by adjusting its vertical positioning.

The padding & the font size was discussed and agreed to constitute an improvement with @kyleaparker and it comes after a suggestion from @twstokes.



### To Test

1. Start creating a new site using the app (From My Site screen tap the ▼ icon next to the site title)
2. On the Site Picker screen tap on the ➕ menu button
3. Select **Create WordPress.com site**
4. Choose any design or tap the **SKIP** menu button
5. Type `_` or any word with non-English characters
6. **Verify** that the text message under the input is not reaching the edges of the screen & it's smaller than it originally was.
7. Rotate the device to landscape orientation
8. **Verify** that the text message is visible when the keyboard is displayed (originally it was hidden behind the keyboard)
7. Rotate the device back to portrait orientation
8. **Verify** that the vertical positioning of the text message is the same as it was at step 6 (approximately at the center when the keyboard is visible)

### Preview

| Before | After |
| --- | --- |
| <img src="https://user-images.githubusercontent.com/4588074/158990869-50644126-aee4-4048-b491-994970c06683.png" width="314"> | <img src="https://user-images.githubusercontent.com/4588074/158995454-caca2203-a38d-4da4-b6e3-39128d928cb6.png" width="314"> |

<details>
<summary><h4>Landscape</h4></summary>

| Before |
| --- |
| <img src="https://user-images.githubusercontent.com/4588074/158991223-0e94605f-81fe-42cf-b7b0-7062201b8145.png" width="628"> |


| After |
| --- |
| <img src="https://user-images.githubusercontent.com/4588074/158990518-6d92317a-21e0-4e87-a333-92c0d6ef6754.png" width="628"> |

</details>

---

### Regression Notes
1. Potential unintended areas of impact
N/A
2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A
3. What automated tests I added (or what prevented me from doing so)
N/A

#### PR submission checklist:


#### PR reviewer notes:

- One reviewer approval should be enough to merge this.
- No release notes are added because the main changes (PR #17985) for this will land in `19.5`
